2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/sinks/NsqSink.scala

```diff
@@ -0,0 +1,60 @@
+/*
+* Copyright (c) 2013-2017 Snowplow Analytics Ltd.
+* All rights reserved.
+*
+* This program is licensed to you under the Apache License Version 2.0,
+* and you may not use this file except in compliance with the Apache
+* License Version 2.0.
+* You may obtain a copy of the Apache License Version 2.0 at
+* http://www.apache.org/licenses/LICENSE-2.0.
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the Apache License Version 2.0 is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied.
+*
+* See the Apache License Version 2.0 for the specific language
+* governing permissions and limitations there under.
+*/
+package com.snowplowanalytics.snowplow
+package collectors
+package scalastream
+package sinks
+
+// NSQ
+import com.snowplowanalytics.client.nsq.NSQProducer
+
+// Scala
+import scala.collection.JavaConverters._
+
+// This project
+import model._
```

Comment:
could you reformat the imports like in [CollectorService](https://github.com/snowplow/snowplow/blob/662b48648d984930017d86be6e2e061ba01f6260/2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala)

- java imports
- scala imports
- other libraries imports
- snowplow-related imports